src/Traits/Encodable.php

```diff
@@ -79,9 +79,6 @@ public function hex(string $mode = Config\Hex::ENCODE) : self
                     return mb_chr(hexdec($matched[1]));
                 }, $this->string);
                 break;
-
-            default:
-                throw new RuntimeException('Invalid mode');
```

Comment:
I understand the reasoning behind this change but I disagree with it. If you remove this change the rest is good and I'll merge it in.

Reply:
I will revert this change and you can figure out the correct approach to do this.